packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/views/diagramfilter/services/BrightenButtonDescriptionProvider.java

```diff
@@ -0,0 +1,74 @@
+
+import java.util.List;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.diagrams.dto.FadeDiagramElementInput;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.diagrams.Diagram;
+import org.eclipse.sirius.components.forms.ButtonStyle;
+import org.eclipse.sirius.components.forms.WidgetIdProvider;
+import org.eclipse.sirius.components.forms.description.ButtonDescription;
+import org.eclipse.sirius.components.representations.VariableManager;
+import org.eclipse.sirius.web.application.views.diagramfilter.services.api.IDiagramFilterActionContributionProvider;
+import org.springframework.stereotype.Service;
+
+/**
+ * Provides the description of the brighten button for the diagram filter's split button.
+ *
+ * @author gdaniel
+ */
+@Service
+public class BrightenButtonDescriptionProvider implements IDiagramFilterActionContributionProvider {
+
+    private IObjectService objectService;
+
+    private DiagramFilterHelper diagramFilterHelper;
+
+    public BrightenButtonDescriptionProvider(IObjectService objectService, DiagramFilterHelper diagramFilterHelper) {
+        this.objectService = objectService;
+        this.diagramFilterHelper = diagramFilterHelper;
+    }
+
+    @Override
+    public ButtonDescription getButtonDescription() {
+        return ButtonDescription.newButtonDescription("diagram-filter/split-button/brighten")
+                .idProvider(new WidgetIdProvider())
+                .targetObjectIdProvider(variableManager -> variableManager.get(VariableManager.SELF, Object.class).map(this.objectService::getId).orElse(null))
+                .labelProvider(variableManager -> "Brighten")
+                .iconURLProvider(variableManager -> List.of())
+                .isReadOnlyProvider(variableManager -> false)
+                .buttonLabelProvider(variableManager -> "Brighten")
+                .imageURLProvider(variableManager -> "/icons/full/obj16/BrightenTool.svg")
+                .pushButtonHandler(variableManager -> {
+                    var diagram = variableManager.get("diagram", Diagram.class).get();
+                    var nodeIds = this.diagramFilterHelper.getSelectedElementIds(variableManager);
+                    var editingContext = variableManager.get("editingContext", IEditingContext.class).get();
+                    return this.diagramFilterHelper.sendDiagramEvent(variableManager, new FadeDiagramElementInput(UUID.randomUUID(), editingContext.getId(), diagram.getId(), nodeIds, false));
+                })
+                .diagnosticsProvider(variableManager -> List.of())
+                .kindProvider(variableManager -> "")
+                .messageProvider(variableManager -> "")
+                .styleProvider(variableManager -> ButtonStyle.newButtonStyle()
+                        .backgroundColor("#ffffff")
+                        .foregroundColor("#000000")
```

Comment:
Do you need to change the default style of the buttons? Because `#000000` is not the color of the text in Sirius Web and in most applications based on Sirius Web.

Reply:
@mbats asked me to put the background in white, because the default colors for the buttons weren't working fine (see screenshot below).

![image](https://github.com/eclipse-sirius/sirius-web/assets/6516281/90d00116-63a3-41af-8fe7-754209eb361e)


The default foreground color for buttons is white, so I had to change it to make the text visible. I updated it to use `#261E58` to match the default text color in Sirius Web, but it will be hardcoded and won't change if the theme changes. Let me know if this is fine for you or if we need something else

Reply:
It's ok to be hardcoded to the Sirius Web theme for now. I'll just update the label of this feature because it has nothing to do with making things brighter :)